docusaurus/docs/get-started/folder-structure.md

```diff
@@ -11,7 +11,7 @@ keywords:
 sidebar_position: 2
 ---
 
-After you install the `create-plugin` tool and have answered the prompts, your project should look similar to this:
+After you [install the `create-plugin` tool](./get-started.mdx#step-1-scaffold-a-new-plugin) and have answered the prompts, your project should look similar to this:
```

Comment:
maybe just have the link on install? The mixed fonts make this look a bit jarring? (we could also investigate this further as a follow up)
<img width="259" alt="Screenshot 2023-10-26 at 19 00 10" src="https://github.com/grafana/plugin-tools/assets/19860021/9b57192b-cc11-487a-a6db-374f9cf92ba0">

Reply:
done.